node-require/calculate.js

```diff
@@ -0,0 +1,24 @@
+const add = require('./add');
+const divide = require('./divide');
+const multiply = require('./multiply');
+const subtract = require('./subtract');
+
+const num1 = process.argv[2];
+const num2 = process.argv[4];
+const op = process.argv[3];
+
+function testing(num1, op, num2) {
+  const parsed = parseInt(num1);
+  const parsed2 = parseInt(num2);
```

Comment:
Your `calculate.js` should also calculate correct result for numbers with decimal places (ex: `3.15 plus 6.23`).